### PR TITLE
example correction

### LINKED ISSUE #346: Error in Example Code
The example code at [https://carlolucibello.github.io/GraphNeuralNetworks.jl/stable/](https://carlolucibello.github.io/GraphNeuralNetworks.jl/stable/) has a small problem:

```julia
[...]
for epoch in 1:100
    [...]
    @info (; epoch, train_loss=loss(train_loader), test_loss=loss(test_loader))
end
```

There is no function signature for `loss(dl::DataLoader)`.  The fix is obviously simple: add the required `model` arg--PR inbound 😉

I could also make a PR to implement CI testing on the docs examples, if that would be welcome.  I like doing that for my packages, since this kind of thing has a way of popping up.

### PR DESCRIPTION
**Pull request by eahenle**
fixes #346 by adding the missing `model` argument to calls to `loss`

### CODE REVIEW

**Comment by CarloLucibello** [comment]
whoops, thanks!
